google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/wrappers/AbstractMutation.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.wrappers;
+
+import com.google.api.core.InternalApi;
+import com.google.bigtable.v2.Mutation;
+import com.google.bigtable.v2.Mutation.DeleteFromColumn;
+import com.google.bigtable.v2.Mutation.DeleteFromFamily;
+import com.google.bigtable.v2.Mutation.DeleteFromRow;
+import com.google.bigtable.v2.Mutation.SetCell;
+import com.google.cloud.bigtable.data.v2.wrappers.Range.TimestampRange;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.ByteString;
+import java.util.List;
+import javax.annotation.Nonnull;
+
+/** This is the base DSL for creating mutations for {@link RowMutation} */
+@InternalApi
```

Comment:
This class can't be `@InternalApi` because it has public methods and the public class `RowMutation` extends it.

Reply:
Changed it to InternalExtensionOnly